jina/flow/base.py

```diff
@@ -110,6 +110,7 @@ def __init__(
         daemon: Optional[bool] = False,
         default_swagger_ui: Optional[bool] = False,
         description: Optional[str] = None,
+        dump_path: Optional[str] = '',
```

Comment:
So there's no way to have dump_path be preserved in the args without adding it here? It might be seen as polluting the namespace, with something that is very executor-specific

Reply:
We can hide it using `SHOW_ALL_ARGS`. Let me change that.

Reply:
Removed using `SHOW_ALL_ARGS`